Earthfile

```
@@ -130,3 +128,30 @@ test-integration:
 test:
   BUILD +test-unit
   BUILD +test-integration
+
+# -----------------------Building-----------------------
+
+build:
+  COPY +go/translate translate
+  WORKDIR translate
+  RUN \
+  --mount=type=cache,target=/go/pkg/mod \
+  --mount=type=cache,target=/root/.cache/go-build \
+    CGO_ENABLED=0 go build -o translate cmd/translate/main.go
```

It keeps it cleaner to set env variables using ENV.

```suggestion
  ENV CGO_ENABLED=0 
  RUN \
  --mount=type=cache,target=/go/pkg/mod \
  --mount=type=cache,target=/root/.cache/go-build \
     go build -o translate cmd/translate/main.go
```